extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -120,28 +124,39 @@ class AmpStickyAd extends AMP.BaseElement {
   }
 
   /**
-   * The listener function that listen on onScroll event and
-   * show sticky ad when user scroll at least one viewport and
-   * there is at least one more viewport available.
+   * The listener function that listen on viewport scroll event.
+   * And decide when to display the ad.
    * @private
    */
-  displayAfterScroll_() {
+  onScroll_() {
+    if (isExperimentOn(this.win, TAG)) {
+      this.displayAfterScroll_();
```

Comment:
`#deferMutate` will only helps to mutate callback to execute at earliest point. But resources manager won't measure elements until scroll velocity decrease to certain threshold. so i think it's fine to use `this.viewport_.onScroll()` as resources manager already handle the optimization for me. 

I am also OK with using `this.viewport_.onChanged()` as on changed will only be fired after scroll slows down.  
https://github.com/ampproject/amphtml/blob/master/src/service/viewport-impl.js#L713